### ISSUE TITLE
Help with 20.09 release with helping to write release notes and release

### ISSUE THREAD

**Issue by garbas** [NixOS/nixos-homepage#448]

**Comment by worldofpeace** [comment]
I believe the items to this have now changed since we last talked. We should talk again very soon with Jon included :grin:

**Comment by garbas** [comment]
I've updated the description and added the todo from our previous conversation.

**Comment by jonringer** [comment]
related: https://github.com/NixOS/nixpkgs/issues/95765

**Comment by davidak** [comment]
Yeah, the gnome iso. I think that is a big change for end-users or new users.
Many "journalists" call NixOS a KDE-based distro... so we can say that we now have a **GNOME Live-Image**, but also state that **you can configure 37 Window Managers and 14 Desktop Environments**. This is a big thing that many might not now!!!

https://search.nixos.org/options?query=windowManager.%20.enable&from=0&size=30&sort=relevance&channel=20.09
https://search.nixos.org/options?query=desktopManager.%20.enable&from=0&size=30&sort=relevance&channel=20.09

And as picture, we can use GNOME default desktop, maybe some apps opened like Blender, a Video Player, Steam

**Comment by worldofpeace** [comment]
Yeah, and we also just got cinnamon so there's a lot of DE changes to show off. cc @mkg20001

**Comment by worldofpeace** [comment]
> Many "journalists" call NixOS a KDE-based distro... so we can say that we now have a GNOME Live-Image, but also state that you can configure 37 Window Managers and 14 Desktop Environments. This is a big thing that many might not now!!!

Exactly, when I first used nixos I was like "is there only kde.." and was pleasantly surprised because it's like everything is here.

**Comment by davidak** [comment]
Here are some images of a GNOME desktop in 1080p. Is there one you like?

![Screenshot from 2020-10-11 09-58-14](https://user-images.githubusercontent.com/91113/95676669-0296dd80-0bc0-11eb-8ce0-d4e2199576af.png)
![Screenshot from 2020-10-11 10-07-26](https://user-images.githubusercontent.com/91113/95676670-032f7400-0bc0-11eb-90b5-a14efd7400f1.png)
![Screenshot from 2020-10-11 10-15-47](https://user-images.githubusercontent.com/91113/95676671-03c80a80-0bc0-11eb-9a99-f178781802fc.png)
![Screenshot from 2020-10-11 10-23-50](https://user-images.githubusercontent.com/91113/95676672-04f93780-0bc0-11eb-9946-9b538e8c50b9.png)
![Screenshot from 2020-10-11 10-24-13](https://user-images.githubusercontent.com/91113/95676673-06c2fb00-0bc0-11eb-8a1c-84181c63d369.png)
![Screenshot from 2020-10-11 10-24-33](https://user-images.githubusercontent.com/91113/95676674-088cbe80-0bc0-11eb-8f79-d15bac3972d0.png)
![Screenshot from 2020-10-11 10-28-40](https://user-images.githubusercontent.com/91113/95676676-09bdeb80-0bc0-11eb-85c1-b5603e874698.png)
![Screenshot from 2020-10-11 10-30-20](https://user-images.githubusercontent.com/91113/95676677-0aef1880-0bc0-11eb-9e5e-f8ec0193e806.png)

**Comment by jonringer** [comment]
active jobsets are determined https://github.com/NixOS/nixos-org-configurations/blob/master/channels.nix

we can use that to list current versions of nixpkgs manual

**Comment by jonringer** [comment]
Semi-unrelated, it might also be nice to have the default wallpaper be release related, like the 20.09 nightingale logo, and unstable will just have the undecorated NixOS logo.

It will also look good for distrowatch where there's some correlation between a particular branch and the wallpaper shown (kind of like people have nostalgia around an Ubuntu wall paper)

**Comment by worldofpeace** [comment]
@jonringer Yep, I believe we talked about this in a GO/NO-GO for future criteria for NixOS on the artwork side

**Comment by jonringer** [comment]
just ` update the release team presentation at https://nixos.org/teams/nixos_release.html, after the redesign` needs to be done

Going to make this as done in the blocker project

**Comment by garbas** [comment]
I also created this issue https://github.com/NixOS/nixos-search/issues/217
